src/uipath/_cli/cli_eval.py

```diff
@@ -138,7 +137,9 @@ def eval(
         asyncio.run(console_reporter.subscribe_to_eval_runtime_events(event_bus))
 
         try:
-            runtime_factory = generate_runtime_factory()
+            # Last one is the Python script runtime factory
+            runtime_factory = Middlewares._runtime_factories[-1]
```

Comment:
This does not have the fallback logic..

Know what i just realized?
```
class FallbackRuntimeFactory(UiPathBaseRuntimeFactory):
  def __init__(self, ..):
    self. factories = []

  def from_context(...):
    for factory in self.factories:
       try:
          return factory.from_context(..)
       except:
          ....

  def discover_...
    # this does the merging logic..
```

Reply:
What I mean is, if someone imports langgraph library but doesn't define a graph,`Middlewares._runtime_factories[-1]` will end up using the langgraph factory and will never fallback to the script runtimefactory.